### DISCUSSION TITLE
disable updated after specific versions

### DISCUSSION THREAD

**Discussion by satanTime** [renovatebot/renovate#8080] [Request Help]
Hi there,

I have [a rule](https://github.com/satanTime/ng-mocks/blob/master/renovate.json#L509-L513):
```
    {
      "paths": ["+(e2e/a11/package.json)"],
      "packageNames": ["rxjs"],
      "allowedVersions": "<=6.5"
    },
```

and I expect no updates to `6.6.3` for `e2e/a11/package.json`, but a related PR has been created: https://github.com/satanTime/ng-mocks/pull/342

I think I'm doing something wrong, might you correct me?

Thanks in advance!

**Comment by rarkins** [comment]
The path syntax doesn't appear to be matching. Reproduced: https://runkit.com/embed/hki327y4502r

**Reply by satanTime** [reply]
Yeah, but that's based on the documentation: https://take.ms/mKxJo
Also if I run the code right on runkit it says `true`: https://runkit.com/embed/36zaka6iz3bx

**Reply by satanTime** [reply]
I've removed the wrapper of `+()`, and now it works
